bin/reth/src/main.rs

```diff
@@ -39,6 +39,10 @@ pub struct EngineArgs {
     /// Configure the target number of blocks to keep in memory.
     #[arg(long = "engine.memory-block-buffer-target", conflicts_with = "legacy", default_value_t = DEFAULT_MEMORY_BLOCK_BUFFER_TARGET)]
     pub memory_block_buffer_target: u64,
+
+    /// Enable state root task
+    #[arg(long = "engine.state-root-task", conflicts_with = "legacy", default_value = "false")]
```

Comment:
default is false already
```suggestion
    #[arg(long = "engine.state-root-task", conflicts_with = "legacy")]
```